src/main/java/com/dougnoel/sentinel/elements/WindowsElement.java

```diff
@@ -244,9 +228,61 @@ public Color getColor() {
 	@Override
 	public boolean attributeEquals(String attribute, String value) {
 		if (attribute.equalsIgnoreCase("color")) {
-			var hexColor = getColor().asHex();
+			var hexColor = getColorAtOffset().asHex();
```

Comment:
It's actually used in another area or two now, and by returning it as a color instead of hex we're able to use the selenium color converter to convert it between selenium and java color objects.

A change I want to make is to create a java color object, actually, rather than a selenium color, though I think that also makes conversion to hex harder. So it's something to discuss once we implement the image comparison in a different area.